### ISSUE TITLE
css not found in wasm 5.0

### ISSUE THREAD

**Issue by Jakeharlan73** [Blazored/Toast#129]
Using Blazor .NET 5.0 and app is not finding the _content folder.  I've seen this issue raised before, but no instruction on how to resolve.  https://github.com/Blazored/Toast/issues/39

The samples you provide work, and the only difference I can see is I am using .NET 5.0.  
Is this a known limitation?  If so, do you have a workaround or do you have plans to fully support .NET 5.0?

**Comment by chrissainty** [comment]
Hi Jake, 

The package works fine on .NET 5. This usually is the result of either a caching issue with the browser or forgetting to reference our stylesheet in your host page.

**Comment by behluluysal** [comment]
I have faced with the same problem. It's not about cache. When you use min.css the css is not loading properly. I used unminifed file and everything works like a charm. I hope this will help you @chrissainty

**Comment by Jakeharlan73** [comment]
I have not gotten as far a publishing.
I did a cold boot to ensure an IIS service wasn't held over.  I then created both a 2.1 and 5.0 Blazor project and successfully imported the Toast unit.  
As a further test of a caching issue I created both a .css and .js file for testing.  I made minor tweaks and restarted the app; The changes take affect every time.    Not sure where/when Blazor is getting trapped in a caching issue but it does appear to be related.
